src/components/localization/language-switcher.js

```diff
@@ -41,14 +41,14 @@ const LanguageSwitch = ({ i18n, is_high_nav, short_name }) => {
         const { id } = e.target
         const current_lang = language || 'en'
         const path = id === '/en/' ? '/' : id
-
+        
         if (`/${current_lang}/` !== id) {
             const current_path = window.location.pathname
             const current_hash = window.location.hash
             const destination_path = `${path}${
                 current_lang === 'en'
                     ? current_path.replace(/\//u, '')
-                    : current_path.replace(/\/.+?\//u, '')
+                    : current_path.replace(/\/.+?(\/)|(\/\w+)/u, '')
```

Comment:
Can we have a comment here to explain what the regex does please 🙏

Reply:
Created a variable with a self explanatory name.
const path_with_or_without_slash = /\/.+?(\/)|(\/\w+)/u

Reply:
Created a variable with a self explanatory name.
const path_with_or_without_slash = /\/.+?(\/)|(\/\w+)/u